### DISCUSSION TITLE
How to remove the Image placeholder?

### DISCUSSION THREAD

**Discussion by Ananthumpillai** [GrapesJS/grapesjs#5876] [General]
Is there a way to remove the image placeholder from the canvas.

If we drop a image element and cancel the modal this image placeholder is not getting removed.
![Image](https://github.com/GrapesJS/grapesjs/assets/138502626/f677d3c1-0c23-42cf-86bf-67623632b26e)

**Comment by artf** [comment]
This is the [default component image](https://github.com/GrapesJS/grapesjs/blob/53c4b9ee3a00b31b0ba2e8f5224f30f33815c544/src/dom_components/model/ComponentImage.ts#L23). You can extend the `image` component and update the `src` property.
